src/diagnostics.rs

```diff
@@ -261,6 +268,7 @@ fn report_msg<'tcx>(
     diag_level: DiagLevel,
     title: &str,
     span_msg: String,
+    labels: Vec<String>,
```

Comment:
`span_msg` and `labels` are referring to the same thing, aren't they?

Reply:
No, sadly. This is my hack to get the line break that I want into the span message, so that they look like this:
```
5 |         let _z = &mut *y;
  |                  ^^^^^^^
  |                  |
  |                  trying to reborrow alloc953+0x0 with Unique permission, but parent tag <1981> only grants SharedReadOnly permission for this memory
  |                  this is a reborrow of alloc953[0x0..0x4]
```
If I just add a newline into `span_msg`, I end up with this:
```
5 |         let _z = &mut *y;
  |                  ^^^^^^^
  |                  |
  |                  trying to reborrow alloc953+0x0 with Unique permission, but parent tag <1981> only grants SharedReadOnly permission for this memory
this is a reborrow of alloc953[0x0..0x4]
  |                  
```

Reply:
What I meant is, they could become a single `Vec<String>` -- `span_msg` is treated like the labels are?

I called it `span_msg` back then since the term "label" meant nothing to me -- it did not say where the label was put.

Reply:
Oh! I see. Adjusted.